common/utils/src/main/resources/error/error-classes.json

```diff
@@ -1860,6 +1860,11 @@
         "message" : [
           "WITHIN GROUP is required for inverse distribution function."
         ]
+      },
+      "WRONG_NUM_ORDERINGS" : {
+        "message" : [
+          "WITHIN GROUP requires <expectedNum> orderings but the actual number is <actualNum>."
```

Comment:
can we mention the function name?

Reply:
The parent error class contains it.
`"Invalid inverse distribution function <funcName>."`